t/08-autotest.t

```diff
@@ -59,7 +59,7 @@ my $mock_basetest = Test::MockModule->new('basetest');
 $mock_basetest->noop('_result_add_screenshot');
 # stop run_all from quitting at the end
 my $mock_autotest = Test::MockModule->new('autotest', no_auto => 1);
-$mock_autotest->noop('_exit');
+$mock_autotest->noop('_terminate');
```

Comment:
As asked in the daily, would be nice if you could extend the comment to also say that we can't use coverage because it's not a fork